test/Microsoft.AspNet.Identity.EntityFramework.Test/DefaultPocoTest.cs

```diff
@@ -65,6 +67,152 @@ public void DropDatabaseStart()
             IdentityResultAssert.IsSuccess(await userManager.DeleteAsync(user));
         }
 
+        [Fact]
```

Comment:
These are new ones I added to test if navigation properties work when used with `Include` on dbContext